deploy/docker/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apk add --no-cache \
     autoconf \
     automake \
     bash \
+    bison \
```

Comment:
change the base image at line 1 instead ?

Reply:
in fact, I wonder why there is to be `RUN apk add` at all in this docker file.
it was probably a quick fix because the base image lacked some packages.